### DISCUSSION TITLE
Mais

### DISCUSSION THREAD

**Discussion by AlvaroMbeiaDanie1234** [flet-dev/flet#1747] [Ideas]
Caro programadores eu gostaria que o flet também tivesse alguns funcionalidades para fazer desenvolvimento de navegadores assim como a biblioteca PyQt5 do python

**Comment by FeodorFitsner** [comment]
Could you post it in English please?

**Reply by Tiago283** [reply]
Dear programmers, I would like flet to also have some features for browser development like the PyQt5 library in Python.

**Reply by FeodorFitsner** [reply]
OK, what are those features?
